src/app/pages/Buy/Crypto/Exolix/hooks/useCurrenciesCount.hook.ts

```diff
@@ -0,0 +1,18 @@
+import { useEffect, useState } from 'react';
+
+import { getCurrenciesCount } from '../exolix.util';
+
+export const useCurrenciesCount = () => {
+  const [currenciesCount, setCurrenciesCount] = useState('');
+
+  useEffect(() => {
+    (async () => {
```

Comment:
extract to separate function, try to avoid using async self-calling anonymous functions in useEffect